### ISSUE TITLE
training on ActivityNet v1.2

### ISSUE THREAD

**Issue by sunsiyang2015** [sujoyp/wtalc-pytorch#7]
hi, when I train on ActivityNet v1.2. the I got mIOU is nan.
how to solve the problem?

**Comment by sujoyp** [comment]
Not sure if this is resolved. Could you please try backtracking through the code to see what is causing the problem? I do not recall getting NaNs in mAP.

**Comment by yhZhai** [comment]
Hi, have you solved this issue? I encountered the same problem, particularly, the detection IoU and training loss became nan after about 1000 iterations. And I used the following parameter to run:
`--dataset-name ActivityNet1.2 --num-class 100 --max-seqlen 3000 --model-name anet1.2 --batch-size 16`

**Comment by yhZhai** [comment]
> hi, when I train on ActivityNet v1.2. the I got mIOU is nan.
> how to solve the problem?

Hi, I solved this problem. This is because thumos14 used validation set to train, while activitynet used training set to train. So you have to change the subset name in _detectionMAP.py_ (maybe some other files).
